enhancements/oc/cli_manager.md

```diff
@@ -0,0 +1,137 @@
+---
+title: cli-manager
+authors:
+  - "@sallyom"
+reviewers:
+  - TBD
+  - "@soltysh"
+approvers:
+  - TBD
+  - "@soltysh"
+creation-date: 2019-12-03
+last-updated: 2019-12-03
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+---
+
+# cli-manager
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+Will CLI manager be supported in disconnected environments?  Currently, we provide disconnected environments 
+cli-artifacts and oc download links that do not require anything outside their cluster. What tools/plugins
+need to be managed in disconnected? 
+
+What CLIs will be managed?    
+Where will binaries be published?    
+[Signing binaries for macOS Catalina](https://developer.apple.com/news/?id=09032019a)    
+[related to above, Go toolchain issue with macOS Catalina](https://github.com/golang/go/issues/34986)    
+
+## Summary
+
+This document is a proposal for a component to manage various tools available for OpenShift and related services.
+A CLI manager is desired so that a user can discover, install and upgrade tools easily and from a single
```

Comment:
s/single location/central location
With what I have in mind you'll be reaching some kind of "central repository" where producers, those willing to publish their binaries, will register their artifacts. And consumers will be able to view and download them.

Reply:
I think it would be beneficial to state the roles at the beginning of this document, producer and consumer are the first that come to my mind.

Reply:
Central repository will be a next one, or a different wording but expressing this intent.

Reply:
Producers here will be tekton, knative, etc.
Consumers will be oc, web console to start with.

Reply:
You need to make sure to mention web console, they will use the exact same mechanism for serving the binaries.